### DISCUSSION TITLE
Get_stocks doesn't return a full list

### DISCUSSION THREAD

**Discussion by tesla897** [alvarobartt/investpy#363] [Q&A]
First, thank you for you work on this package. 

I tried running the get_stocks method:
 stocks_df = investpy.get_stocks(country='india') 

It gives me 1701 symbols from investing.com

but when I look through the entries I see lots of stocks missing. For example, DEEE:
https://in.investing.com/equities/deep-industries-ltd

It doesn't appear in the above list.

And there lot more symbols missing.

**Comment by hydrastarmaster** [comment]
I checked the case, and the same problem is present for other libraries that work on symbol extraction from Google and Yahoo. In the NYSE case, count just 467 symbols. Note that on Investing site the detail pages are present. The problem is just in the extraction of the symbols list.
